docs-client/src/containers/App/index.tsx

```diff
@@ -389,6 +445,36 @@ class App extends React.PureComponent<Props, State> {
     });
   };
 
+  private httpMethodClass = (httpMethod: string) => {
```

Comment:
Is it possible to return `httpMethodCommon` and `httpMethod*` here so the caller doesn't have to add `httpMethodCommon`?